Reaper/Reaper64.download.recipe

```diff
@@ -25,7 +25,7 @@
 				<key>url</key>
 				<string>%DOWNLOAD_URL%</string>
 				<key>re_pattern</key>
-				<string>(?P&lt;url&gt;files\/\d\.x\/reaper(?P&lt;version&gt;\d+\.*)_x86_64\.dmg)</string>
+				<string>(?P&lt;url&gt;files\/\d\.x\/reaper(?P&lt;version&gt;\d+\.*(|.))_x86_64\.dmg)</string>
```

Comment:
```suggestion
				<string>(?P&lt;url&gt;files\/\d\.x\/reaper(?P&lt;version&gt;\d+\w?)_x86_64\.dmg)</string>
```
Can I suggest the above regex instead? It avoids creating another capturing group and should be reasonably robust, that is until Cockos completely change the naming scheme again! I've also removed the `\.*` – this appeared in a previous PR but I didn't spot it at the time and am not sure what purpose it serves... it's certainly not relevant with their current naming scheme.

Reply:
That looks better - my regex-fu isn't as refined as yours - still don't know a lot of the tokens.